textile/chat-features.textile

```diff
@@ -957,9 +1008,9 @@ h3(#chat-structs-typing-event-V3). Typing Event V3
 
 <pre>
   {
-    "type": "typing.set.changed",
+    "type": TypingSetEventType.SetChanged
```

Comment:
This is supposed to be string value right? Why is it a code instead?

Reply:
I guess we have assigned similar enum for others too 👍

Reply:
Yeah - tried to make it match the enums so its clear